### PR TITLE
Grid component

### PR DESCRIPTION
**Pull request by i-am-a-codegirl**
Please review my grid components

this page contains the grid component

live link: https://css-companion.netlify.app/components/grid.html

Following are the features of this page

- nav: contains brand name and links to the home page and components library.
- main: contains the documentation of button components with snippets and examples.
- aside: contains links to all my component libraries.
- footer: contains links to my social media

image
image

### CODE REVIEW

**Review by zabihshaik — COMMENTED** [review]
Looks good. Few suggestions:
The Grid-3 should have all 3 items in 1 row.
Use Prettier for formatting the code.
Remove unnecessary blank spaces before pushing the code.
Use meta tag in head of html file.

File: component/grid/grid.css [hunk 1]
```diff
@@ -0,0 +1,547 @@
...
+    border: 1px solid var(--secondary-color);
+    margin: 0.5rem;
+    height: 3rem;
+    border-radius: 0.5rem;
+}
+
+.grid-3 {
+    display: grid;
+    grid-template-columns: 1fr;
+
```
Comment (nyyshaaa) [inline component/grid/grid.css]: i think here you gave 1 fr that's why they went to next lines. as @zabihshaik  mentioned for column grids they would have probably came under one row.

**Review by nyyshaaa — APPROVED** [review]
just one suggestion if you wish to implement. all else looks clean to me .

File: component/grid/grid.css (hunk 1 above)
Comment (i-am-a-codegirl) [reply component/grid/grid.css]: oh thanks

**Review by khushijohri2001 — APPROVED** [review]
UI looks cool

**Pull request merged**